### ISSUE TITLE
Additional processes need to be added to Accessibility blacklist

### ISSUE THREAD

**Issue by sjthespian** [Hammerspoon/hammerspoon#1964]
I'm seeing accessibility warnings from a handful of apps:

2018-10-16 09:25:41: 09:25:41 ** Warning:   wfilter: No accessibility access to app Todoist Networking (no watcher pid)
2018-10-16 09:25:41: 09:25:41 ** Warning:   wfilter: No accessibility access to app Safari Storage (no watcher pid)
2018-10-16 09:25:41: 09:25:41 ** Warning:   wfilter: No accessibility access to app Todoist Database Storage (no watcher pid)
2018-10-16 09:25:53: 09:25:53 ** Warning:   wfilter: No accessibility access to app AAM Updates Notifier (no watcher pid)
2018-10-16 09:25:53: 09:25:53 ** Warning:   wfilter: No accessibility access to app Slack Helper (no watcher pid)

**Comment by AlbertVilaCalvo** [comment]
You can add `hs.window.filter.ignoreAlways['Sourcetree Networking'] = true` to fix this.

See http://www.hammerspoon.org/docs/hs.window.filter.html#ignoreAlways
